tests/functional/pages/surveys/checkbox_other/0/pizza-toppings.page.js

```diff
@@ -0,0 +1,17 @@
+import QuestionPage from '../../question.page'
+
+class PizzaToppingsPage extends QuestionPage {
+
+  selectOther() {
+      browser.click('input[id="ca3ce3a3-ae44-4e30-8f85-5b6a7a2fb23c-6"]');
+      return this;
+  }
+
+  inputOtherValue(text) {
```

Comment:
I think `setOtherValue` would be more consistent with other function names that set values